recipes/lmdb/all/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+project(cmake_wrapper)
```

Comment:
The `cmake_minimum_required` was necessary.
The hooks were complaining that the version you selected was too low.